roles/matrix-bot-postmoogle/defaults/main.yml

```
@@ -110,11 +111,20 @@ matrix_bot_postmoogle_noencryption: false
 
 matrix_bot_postmoogle_domain: "{{ matrix_server_fqn_matrix }}"
 
-# in-container port
+# Mandatory TLS, even on plain SMTP port
+matrix_bot_postmoogle_tls_required: false
+
+# in-container ports
 matrix_bot_postmoogle_port: '2525'
+matrix_bot_postmoogle_tls_port: '25587'
 
-# on-host port
+# on-host ports
 matrix_bot_postmoogle_smtp_host_bind_port: '25'
+matrix_bot_postmoogle_smtps_host_bind_port: '587'
```

```suggestion
matrix_bot_postmoogle_submission_host_bind_port: '587'
```

- https://en.wikipedia.org/wiki/SMTPS#Difference_between_SMTPS_and_smtps
- https://en.wikipedia.org/wiki/Message_submission_agent

`smtps` kind of hints towards port `465`.